core/modules/system/system.theme.inc

```diff
@@ -171,19 +171,18 @@ function theme_system_modules_fieldset($variables) {
     // Add the third column: module version number.
     $row[] = array('class' => array('version'), 'data' => backdrop_render($module['version']));
 
-    if ($requires || $required_by) {
-      $requirements .= ' <div class="requirements">';
-      if ($requires) {
-        $requirements .= '<div class="admin-requirements">' . t('Requires: !module-list', array('!module-list' => implode(', ', $module['#requires']))) . '</div>';
-      }
-      if ($required_by) {
-        $requirements .= '<div class="admin-requirements">' . t('Required by: !module-list', array('!module-list' => implode(', ', $module['#required_by']))) . '</div>';
-      }
-      $requirements .= '</div>';
-
-      // Build a requirements toggle.
-      $toggle .= ' <a class="requirements-toggle" href="#">' . t('more') . '</a>';
+    $requirements .= ' <div class="requirements">';
+    if ($requires) {
+      $requirements .= '<div class="admin-requirements">' . t('Requires: !module-list', array('!module-list' => implode(', ', $module['#requires']))) . '</div>';
```

Comment:
Can we please leave all the code-y stuff out of `t()`?:

```suggestion
       $requirements .= '<div class="admin-requirements">' . t('Requires:') . implode(', ', $module['#requires']) . '</div>';
```

Reply:
yes, I suppose module names will not be translated. Thanks!